service/firewall/bypassing.go

```diff
@@ -57,6 +57,10 @@ func PreventBypassing(ctx context.Context, conn *network.Connection) (endpoints.
 
 	// Block bypass attempts using an (encrypted) DNS server.
 	switch {
+	case looksLikeOutgoingDNSRequest(conn) && module.instance.Resolver().IsDisabled():
+		// Allow. Packet will be analyzed and blocked if its not a dns request, before sent.
+		conn.Inspecting = true
```

Comment:
conn.Inspecting is currently unused. Are you using it?

Reply:
Yes Im using it here:
https://github.com/safing/portmaster/pull/1728/files#diff-e0558e79bafb1ddeb52febdec98e2d2ddfd0dedb243d8533e77ff7b365b6a45cR485